documentation/plant-description-engine/plant-description-engine-sosd.md

```diff
@@ -0,0 +1,488 @@
+# Plant Description Engine - System of Systems Description (SosD)
+
+## Abstract
+This document describes how a system of systems that includes a [Plant Description Engine] (PDE) interacts with the PDE.
+
+## Overview
+
+This document describes how a system of systems that includes the Plant Description Engine (PDE) Arrowhead Framework system interacts.
+
+A plant (System of Systems / Local cloud) is assumed to include the following systems:
+ - [Service Registry]
+ - [Authorization]
+ - [Orchestrator]
+ - [Plant Description Engine]
+ - Some Producer systems
+ - Some Consumer systems
+
+We will as an example consider a plant that has the four core systems and six custom systems:
+ - Operations Center that manages and monitors the Plant Description Engine
+ - A that produces service X
+ - B that produces service Y
+ - C that produces service Z
+ - D that consumes services X, Y and Z.
+   This system should always be connected to a system producing X.
+   It should also be connected to either a system that provides Y or a system the provides Z but not both.
+ - Invent system that produces the [Inventory] service.
+
+The four custom systems A-D also provides the [Monitorable] service.
+
+
+## The plant description(s) describing the example system
+
+The basic Arrow Head core systems are connected to each other
+
+IMAGE
+
+This corresponds to a Plant Description with the core systems:
+
+```json
+{
+	"id": 1,
+	"plantDescription": "ArrowHead core",
+	"systems": [
+		{
+			"systemName": "Service Registry",
+			"systemId": "service_registry",
```

Comment:
Please note that if the systemId in PDE has any connection with system's name in Service Registry, then using underscores can be a problem. Since Arrowhead 4.4.0 system's name has to match a particular regexp (see issue #202 or CommonNamePartVerifier class). Because of this we have to rename service_registry to serviceregistry.

Reply:
Fixed, together with a bunch of other invalid system names in the docs.